SASS Styleguide/File organization.md

```diff
@@ -128,12 +128,16 @@ When you're working on a project with `_shame.scss` always check the file when y
 The colors file should only contain color variables of the base and global kind.
 All colors for blocks or elements should go into their local file on the top.
 
-Base colors are actual color values used on the site, they will be used by both global colors and block-elements colors. Syntax and example:
+Base colors are actual color values used on the site, they will be used by both global colors and block-elements colors. 
+
+When naming color variables avoid using names that correspond to the actual colors, better use some thing like `silver`, `wood`, `mercury`, etc...
```

Comment:
Silver is actually a color: https://en.wikipedia.org/wiki/Silver_(color)

So it's best if we remove it or have a different example here

Reply:
👍